core/transport/src/main/proto/grpc/block_worker.proto

```diff
@@ -131,6 +132,43 @@ message CacheRequest {
   optional bool async = 6;
 }
 
+// Request for load a block into alluxio
+// next available id: 4
+message LoadRequest {
+  repeated FileBlocks files = 1;
+  optional string user = 2;
+  optional int64 bandwidth = 3;
+}
+
+message FileBlocks{
+  optional string ufs_path = 1;
+  repeated int64 block_id = 2;
+  required int64 block_size = 3;
+}
+
+message LoadResponse {
+  repeated FileStatus file_status = 1;
+  required TaskStatus status = 2;
+}
+
+message FileStatus {
+  repeated BlockStatus block_status = 1;
+}
+
+enum TaskStatus {
+  SUCCESS = 0;
+  FAILURE = 1;
+  PARTIAL_FAILURE = 2;
+}
+
+message BlockStatus {
+  // The status code, which should be an enum value of [google.rpc.Code][google.rpc.Code].
+  required int32 code = 1;
+
+  // A developer-facing error message
+  optional string message = 2;
```

Comment:
Let's see whether we can directly use Status, if not, we need to define a new proto for this. Only a string won't be enough.